### DISCUSSION TITLE
The MUX_TOKEN_ID environment variable is missing

### DISCUSSION THREAD

**Discussion by TestCK** [muxinc/next-video#260] [Q&A]
Hi,

When I start up I get,

`MuxError: The MUX_TOKEN_ID environment variable is missing or empty; either provide it, or instantiate the Mux client with an tokenId option, like new Mux({ tokenId: 'my token id' }`

Is it mandatory to sign up for Mux to run the project?

**Comment by luwes** [comment]
It's not required. You can change the provider by one of these https://github.com/muxinc/next-video#hosting--processing-providers

**Reply by LahNabil** [reply]
i had the same issue . because i created my .env.local in another directory instead of the root directory
